pyaedt/hfss.py

```diff
@@ -818,7 +823,7 @@ def create_linear_step_sweep(
 
         """
         if sweep_type not in ["Discrete", "Interpolating", "Fast"]:
-            raise AttributeError("Invalid in `sweep_type`. It has to either 'Discrete', 'Interpolating', or 'Fast'")
+            raise AttributeError("Invalid `sweep_type`. It has to be 'Discrete', 'Interpolating', or 'Fast'.")
```

Comment:
@PipKat I am creating a test for this code coverage warning. It will be solved by the beginning of the afternoon.

Reply:
Done